cmd/loom/addressmapper.go

```diff
@@ -102,15 +103,20 @@ func GetMapping() *cobra.Command {
 			if err != nil {
 				return err
 			}
-			out, err := formatJSON(&resp)
-			if err != nil {
-				return err
+			type maxLength struct {
+				From int
+				To   int
 			}
-			fmt.Println(out)
+			ml := maxLength{From: 50, To: 50}
+
+			fmt.Printf("%-*s | %-*s |\n", ml.From, "From", ml.To, "To")
+
+			fmt.Printf("%-*s | %-*s |\n",
+				ml.From, loom.UnmarshalAddressPB(resp.From).String(),
+				ml.To, loom.UnmarshalAddressPB(resp.To).String())
```

Comment:
Table is overkill since there's only one mapping, can just output a single line, e.g.
```
default:0xdeadbeef123 -> eth:0xdeadbeef321
```